pycqed/analysis_v3/helper_functions.py

```diff
@@ -1023,16 +1051,20 @@ def read_analysis_file(timestamp=None, filepath=None, data_dict=None,
         if ana_file is None:
             if filepath is None:
                 if file_id is None:
-                    file_id = '_AnalysisResults'
+                    file_id = 'AnalysisResults'
                 folder = a_tools.get_folder(timestamp)
-                filepath = a_tools.measurement_filename(folder, file_id=file_id)
+                filepath = a_tools.measurement_filename(folder,
+                                                        file_id=f'_{file_id}')
             ana_file = h5py.File(filepath, mode)
         read_from_hdf(data_dict, ana_file)
         if close_file:
             ana_file.close()
     except Exception as e:
         if close_file:
-            ana_file.close()
+            try:
+                ana_file.close()
+            except AttributeError:
+                pass
```

Comment:
If you silently pass an error, it might be good to add a comment that explains why.

Reply:
added now